src/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/table/table.js

```diff
@@ -0,0 +1,175 @@
+import React, { PureComponent, Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiSearchBar,
+  EuiBasicTable,
+  EuiButton,
+  EuiIcon,
+  EuiLink,
+  EuiSpacer,
+} from '@elastic/eui';
+import { getSavedObjectIcon } from '../../../../lib/get_saved_object_icon';
+
+export class Table extends PureComponent {
```

Comment:
@bmcconaghy and I have gone back and forth on this. It's functionally equivalent and I only left it this way because I had originally made this a stateful component and changed it to a pure component.